index.html

```diff
@@ -85,9 +85,295 @@ <h3 class="lc-button-text-color lc-button-text-style" >LET'S CONNECT</h3>
+                            <span>Languages</span>
+                            <span><img class="icon-24px" src="@images/down-sign.svg" alt="down-sign" /></span>
+                        </dt>
+                        <dd>
+                            <ul class="prog-lang-list prog-lang-props prog-lang-layout " >
+                                <li class="flex-box horizontal align-items-center no-bullet">
+                                    <img src="@images/JS_logo.svg" alt="JS logo" />
+                                    <span>JavaScript</span>
+                                </li>
+                                <li class="flex-box horizontal align-items-center no-bullet">
+                                    <img src="@images/HTML-5_logo.svg" alt="HTML 5 logo" />
+                                    <span>HTML</span>
+                                </li>
+                                <li class="flex-box horizontal align-items-center no-bullet">
+                                    <img src="@images/CSS-3_logo.svg" alt="CSS 3 logo" />
+                                    <span>CSS</span>
+                                </li>
+                            </ul>
+                        </dd>
+                        <dt class="tech-menu-head-text-style tech-menu-head-text-layout " id="frameworks" >
+                            <span>Frameworks</span>
+                            <span><img class="icon-24px" src="@images/right-sign.svg" alt="right-sign" /></span>
+                        </dt>
+                        <dd></dd>
+                        <dt class="tech-menu-head-text-style tech-menu-head-text-layout " id="skills" >
+                            <span>Skills</span>
+                            <span><img class="icon-24px" src="@images/right-sign.svg" alt="right-sign" /></span>
+                        </dt>
+                        <dd></dd>
+                    </dl>
+
+                </div>
+
+            </div>
+        </section>
+
+        <!-- Contact Me section -->
+        <section class="flex-box horizontal flex-center transparent-bg black-fg" >
+
+            <!-- Flex box vertical container -->
+            <div class="flex-box vertical flex-center" id="contact-me" >
+            
+                <!-- Contact Me header -->
+                <h1 class="contact-me-header-layout contact-me-header-text-style" id="contact-me-header" >Contact me</h1>
+                
+                <!-- Contact Me text -->
+                <p class="contact-me-text-layout contact-me-text-text-style" id="contact-me-text" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding, 
+I’d love to help with it!</p>
+                
+                <!-- Contact Me contact form -->
+                <form class="flex-box vertical flex-center gap-24px" id="contact-me-form" >
```

Comment:
Fantastic work so far! 😱 However, I believe there is an opportunity for improvement in the background adjustments(means it doesn't responsive) of this particular section. It is crucial to ensure that the background aligns with the Figma design and meets all the project requirements. 

Your current design: 
![con](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/118761303/70a4364e-245c-4d74-8dd7-9db4a6a0e9f0)